apps/full-site-editing/.wp-env.json

```diff
@@ -0,0 +1,6 @@
+{
+	"plugins": [ "./full-site-editing-plugin", "../../../gutenberg" ],
```

Comment:
What do you think about including Jetpack and wpcomsh here so it's closer to Atomic env?

Reply:
We could definitely do that. Do you think it could be a follow up?

One of the reasons that it might not be nice to do this is that every plugin and theme here is the development version. So I'd have to do `npm i` and `npm run build` for every plugin so that it loads correctly. And then I would have to make sure that I do `git pull` regularly. I could see that being a pain with a lot of plugins.

Reply:
> Do you think it could be a follow up?

Sure, that's fine with me.